test/grouping.jl

```diff
@@ -2308,12 +2309,12 @@ end
           combine(gdf, :x => (x -> sum(x)) => :a, :x => (x -> prod(x)) => :b)
     df = DataFrame(g=[1, 1], x=[missing, "a"])
     gdf = groupby_checked(df, :g)
-    @test combine(gdf, :x => sum∘skipmissing => :a, :x => prod∘skipmissing => :b) ==
-          combine(gdf, :x => (x -> sum(skipmissing(x))) => :a, :x => (x -> prod(skipmissing(x))) => :b)
+    @test Matrix(combine(gdf, :x => sum∘skipmissing => :a, :x => prod∘skipmissing => :b)) ==
```

Comment:
Why is `Matrix` needed here? Output names are set explicitly.

Reply:
Ah - right. It was a leftover from earlier design. Fixed